mobile/src/main/res/values/strings.xml

```diff
@@ -157,7 +157,7 @@
     <!-- Background tasks -->
     <string name="send_device_info_to_server">Send device information to server</string>
     <string name="send_device_info_item_prefix">Item name prefix</string>
-    <string name="send_device_info_item_prefix_summary">The Item names of the following features have this prefix</string>
+    <string name="send_device_info_item_prefix_summary">The Item names of the following features have this prefix: %s</string>
```

Comment:
'... have this prefix: %s' -> 'are prefixed with %s'?

Reply:
The first option sounds better if no prefix is set.
Should I add a different string if no prefix is set? "The Item names of the following features have no prefix"

Reply:
Yeah, a separate string makes sense to me. How about 'No prefix is set for the Item names of the following features.'?